flake.nix

```diff
@@ -0,0 +1,58 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/release-22.11";
```

Comment:
Is it worth pinning to a stable release? I usually omit specifying nixpkgs which is equivalent to using unstable by default.

I feel like rust development generally likes to track more cutting-edge versions of tooling, so unstable might be a better bet here. (In either case, the flake lock will pin to a specific revision still.)

Reply:
agreed, force-pushed